script/RumpelConfig.sol

```diff
@@ -1334,7 +1352,90 @@ library RumpelConfig {
+
+    function getSymbioticExpansionBatch3ProtocolGuardConfigs() internal pure returns (ProtocolGuardConfig[] memory) {
+        ProtocolGuardConfig[] memory configs = new ProtocolGuardConfig[](3);
+
+        // ETHFI skipped as reward token - only transfer enabled
+
+        configs[0] = ProtocolGuardConfig({
+            target: MAINNET_SYMBIOTIC_MEV_CAPITAL_WSTETH_COLLATERAL,
+            selectorStates: new SelectorState[](2)
+        });
+        configs[0].selectorStates[0] = SelectorState({
+            selector: ISymbioticDefaultCollateral.deposit.selector,
+            state: RumpelGuard.AllowListState.ON
+        });
+        configs[0].selectorStates[1] = SelectorState({
+            selector: ISymbioticDefaultCollateral.withdraw.selector,
+            state: RumpelGuard.AllowListState.ON
+        });
+
+        configs[1] =
+            ProtocolGuardConfig({target: MAINNET_SYMBIOTIC_SFRXETH_COLLATERAL, selectorStates: new SelectorState[](2)});
+        configs[1].selectorStates[0] = SelectorState({
+            selector: ISymbioticDefaultCollateral.deposit.selector,
+            state: RumpelGuard.AllowListState.ON
+        });
+        configs[1].selectorStates[1] = SelectorState({
+            selector: ISymbioticDefaultCollateral.withdraw.selector,
+            state: RumpelGuard.AllowListState.ON
+        });
+
+        configs[2] = ProtocolGuardConfig({
+            target: MAINNET_SYMBIOTIC_GUANTLET_RESTAKED_SWETH_COLLATERAL,
+            selectorStates: new SelectorState[](2)
+        });
+        configs[2].selectorStates[0] = SelectorState({
+            selector: ISymbioticDefaultCollateral.deposit.selector,
+            state: RumpelGuard.AllowListState.ON
+        });
+        configs[2].selectorStates[1] = SelectorState({
+            selector: ISymbioticDefaultCollateral.withdraw.selector,
+            state: RumpelGuard.AllowListState.ON
+        });
+
+        return configs;
+    }
+
+    function getSymbioticExpansionBatch3TokenGuardConfigs() internal pure returns (TokenGuardConfig[] memory) {
+        TokenGuardConfig[] memory configs = new TokenGuardConfig[](5);
+
+        // underlying
+        // wstETH already added
+        // swETH added in symbiotic-batch-2
+        // ETHFI skipped as reward token
+        configs[0] = TokenGuardConfig({
+            token: MAINNET_SFRX,
+            transferAllowState: RumpelGuard.AllowListState.ON,
+            approveAllowState: RumpelGuard.AllowListState.OFF
+        });
+
+        // collateral tokens
+        configs[1] = TokenGuardConfig({
+            token: MAINNET_SYMBIOTIC_MEV_CAPITAL_WSTETH_COLLATERAL,
+            transferAllowState: RumpelGuard.AllowListState.ON,
+            approveAllowState: RumpelGuard.AllowListState.OFF
+        });
+        configs[2] = TokenGuardConfig({
+            token: MAINNET_SYMBIOTIC_SFRXETH_COLLATERAL,
+            transferAllowState: RumpelGuard.AllowListState.ON,
+            approveAllowState: RumpelGuard.AllowListState.OFF
```

Comment:
In the other two PRs, I think we add the approve function for symbiotic collat tokens too. ([eg](https://github.com/sense-finance/rumpel-wallet/pull/41/files#diff-8ffd9e34c79698015a7836a95eb96407ba8239ca754c7ba1885a2ddf5b4c5245R1460)) any reason for the difference?

Reply:
yes - should be ON. This will allow for users to migrate to new vaults. I'll update.

( not including the newer vaults, which don't have transfer or approve as you mentioned )

Reply:
updated